delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphViews.scala

```diff
@@ -531,7 +544,8 @@ object BlazegraphViews {
                              )
       views                = new BlazegraphViews(agg, eventLog, index, projects, orgs, sourceDecoder)
       _                   <- validateRefDeferred.complete(validateRef(views))
-      _                   <- BlazegraphViewsIndexing.populateCache(config.indexing, views, index).void
+      _                   <- BlazegraphViewsIndexing.deleteNotUsedNamespaces()
```

Comment:
if this is supposed to run asynchronously... it needs to be forked, right?
additionally, if this replays the event log, it should probably be executed along with cache indexing to avoid multiple concurrent replays.

Reply:
As you can see this is not yet implemented. I'll come on the details when I actually implement it.

But no, it doesn't need to be forked. Currently `populateCache` is not being forked and it works, right? Note that in the end it calls `DaemonStreamCoordinator.run`